### PR TITLE
fix: enhance code comments

### PR DESCRIPTION
**Pull request by threewebcode**
Enhance and tidy the code comments

### CODE REVIEW

**Comment by threewebcode** [comment]
@davecgh, Please help review

**Comment by threewebcode** [comment]
@davecgh, Please help review

File: mining/mining.go [hunk 1]
```diff
@@ -520,7 +520,7 @@ mempoolLoop:
 			continue
 		}
 
-		// Fetch all of the utxos referenced by the this transaction.
+		// Fetch all of the utxos referenced by this transaction.
```
Comment (jcvernaleo) [inline mining/mining.go:523]: Good catch here.

File: mining/mining.go [hunk 2]
```diff
@@ -602,7 +602,7 @@ mempoolLoop:
 	blockSigOpCost := coinbaseSigOpCost
 	totalFees := int64(0)
 
-	// Query the version bits state to see if segwit has been activated, if
+	// Query the version bits state to see whether segwit has been activated, if
```
Comment (jcvernaleo) [inline mining/mining.go:605]: I don't think there is any need for this change.  The simpler original version is fine.

**Review by jcvernaleo — CHANGES_REQUESTED** [review]
One change is good, I'd rather you revert the other one.

File: mining/mining.go (hunk 1 above)
Comment (threewebcode) [reply mining/mining.go:523]: Thanks for your feedback.

File: mining/mining.go (hunk 2 above)
Comment (threewebcode) [reply mining/mining.go:605]: Thanks for your opinion and i will discard this change.

**Comment by threewebcode** [comment]
@jcvernaleo The change set is updated with your review. Please take a look again.

**Review by jcvernaleo — APPROVED** [review]
OK

**Pull request merged**